packages/nimble-components/src/card/specs/README.md

```diff
@@ -0,0 +1,114 @@
+treatments, but we are only solving for the Routines use case at this time (grouping related configuration in
+a slide-out). To support other use cases in the future, we could potentially add appearance variants.
+
+We will hold off on adding new design tokens for the `nimble-card` until we get the visual design more settled,
+because we don't know if we will require different tokens from the existing components.
+
+### API
+
+[`fast-card` spec](https://github.com/microsoft/fast/blob/b78c921ec4e49ec9d7ec980f079ec114045df42e/packages/web-components/fast-foundation/src/card/card.spec.md)
+
+-   _Component Name_
+    -   `nimble-card`
+-   _Properties/Attributes_
+    -   _`title-hidden`_
+        -   If set, hides the provided title.
+-   _Methods_
+    -   _(none)_
+-   _Events_
+    -   _(none)_
+-   _CSS Classes and Custom Properties that affect the component_
+    -   _(none)_
+-   _Slots_
+    -   `title`
+        -   A title to display in the `nimble-card`
+    -   `(default)`
+        -   Arbitrary content to display in the `nimble-card`.
+
+### Angular integration
+
+We will create an Angular `nimble-card` directive for the component. The `nimble-card` does not require form integration, so there
+is no need for a ControlValueAccessor.
+
+### Blazor integration
+
+We will create a Blazor wrapper the component and define a `NimbleCard` class. The `nimble-card` component supports child content,
+so it will need a `ChildContent` parameter.
+
+### Additional requirements
+
+-   _User interaction: Do the FAST component's behaviors match the visual design spec? When they differ, which approach is preferable and why?_
+    -   No additional requirements. The initial `nimble-card` will have no interactions.
+-   _Styling: Does FAST provide APIs to achieve the styling in the visual design spec?_
+    -   No additional requirements expected here.
+-   _Testing: Is FAST's coverage sufficient? Should we write any tests beyond Chromatic visual tests?_
+    -   No additional requirements
+-   _Documentation: Any requirements besides standard Storybook docs and updating the Example Client App demo?_
+    -   No additional requirements
+-   _Tooling: Any new tools, updates to tools, code generation, etc?_
+    -   No additional requirements
+-   _Accessibility: keyboard navigation/focus, form input, use with assistive technology, etc._
+    -   If the user supplies a title, we will set `aria-label` to that value.
+    -   The `nimble-card` itself does not receive keyboard focus and tabbing jumps to the first focusable component inside the `nimble-card`.
+-   _Mobile: small screens, touch interactions, mobile-specific integrations_
+    -   No additional requirements. By default, the `nimble-card` fits its height to its content and grows/shrinks horizontally to
+        fit its parent container. Other mobile-friendly considerations are up to the client and the settings on the child components.
```

Comment:
Sounds like this is saying it's default width will be 100%. I think a more expected default would be `auto`, i.e. it'll size to it's contents instead of filling the space of the parent container. i.e. `display: flex` / `display: block` default behavior

Reply:
This was describing the behavior I was seeing with the[ `flex-card` example](https://explore.fast.design/components/fast-card), which I believe is using `display: block`. They set an explicit width in the example, and if you remove the `width` then the card stretches horizontally to fill its parent.

In my prototype, both `display: flex` and `display: block` also make the card stretch horizontally to fill the parent container. If we want the default to be `auto` behavior, I can set `${display('inline-block')}` in the style. Is that what you're suggesting?

Reply:
Ah yea, brain fart, block level elements would try and fill the width.
I was suggestion inline-block but after taking a look at the openui card research page: https://open-ui.org/components/card.research/
looks like most design systems are treating cards as block level elements rather than inline-block, so I'll retract the statement. A block level layout like flex would be good 👍